### PR TITLE
step1: etcher note (fixes #335)

### LINKED ISSUE #335: Failed burning the treehouses image onto the microSD card
### Problem
We should always run with the latest version of Etcher.
If the software has been installed in the past and a new version has been released, burning will failed.

### Steps to reproduce the problem
N/A (If you already have Etcher of old version, You can reproduce.)

### Screenshots
N/A

### Proposed solution
Install to Etcher of new version.

### PR DESCRIPTION
**Pull request by ghost**
Description
Add " _**NOTE:**_ If you run into any issues with flashing, to try and update Etcher."

Raw.Githack preview link
e.g. https://raw.githack.com/sabe-lab/sabe-lab.github.io/sabe-new-branch02/#!pages/vi/pi-setup.md

### CODE REVIEW

**Review by dogi — APPROVED** [review]

**Review by phamduchongan93 — COMMENTED** [review]
remove italic

**Comment by dogi** [comment]
@sabe-lab I am confused ... why did you close this PR?

**Comment by ghost** [comment]
@dogi I'm sorry, I was misunderstanding.

File: pages/vi/pi-setup.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ _**NOTE:**_ The latest image will be at the bottom of the download page and will
     * Click "Flash!".  The process will take approximately 10 minutes to complete.
 1. Once it's done burning, you can view the contents in the `boot` partition of your microSD card using a file explorer.
 1. Unmount and safely eject the microSD card from your computer. 
-
+**NOTE:** If you run into any issues with flashing, to try and update Etcher.
```
Comment (kjong) [inline pages/vi/pi-setup.md:40]: Please make the NOTE formatting consistent with the other notes (ie. blue box).

**Review by kjong — DISMISSED** [review]

File: pages/vi/pi-setup.md (hunk 1 above)
Comment (ghost) [reply pages/vi/pi-setup.md:40]: @kjong Thank you for your advice. I will be more careful.

**Pull request merged**